control-plane/grpc/proto/v1/node/node.proto

```diff
@@ -146,11 +146,41 @@ message DrainState {
   repeated string drain_labels = 2;
 }
 
+message LabelNodeRequest {
+  // Node identification
+  string node_id = 1;
+  // Node label
+  string label = 2;
```

Comment:
how is label meant to be defined at?
Example, single value, multiple comma-separated, etc...

Reply:
multiple semi-colon separated
```
$ kubectl label node node-0-142132 A=B,C=D
error: invalid label spec: A=B,C=D
$ kubectl label node node-0-142132 A=B;C=D
node/node-0-142132 labeled
```

Reply:
Why not have a map of string to string?

Reply:
Done